contracts/token/ERC20/extensions/ERC20Votes.sol

```diff
@@ -24,7 +24,7 @@ import "../../../utils/cryptography/ECDSA.sol";
  *
  * _Available since v4.2._
  */
-abstract contract ERC20Votes is IVotes, ERC20Permit {
+abstract contract ERC20Votes is IERC5805, ERC20Permit {
     struct Checkpoint {
         uint32 fromBlock;
```

Comment:
```suggestion
        uint32 fromTimepoint;
```

Reply:
If we rename here, we will require some instruction for the upgrades plugin.

Reply:
Is this the discussion we've had about the annotation for struct variables?

Reply:
Another argument, there is this function that we have for backward compatibility (with Compound)
```solidity 
function checkpoints(address account, uint32 pos) public view virtual returns (Checkpoint memory)
```

Changing the field name would be modifying the ABI/interface and possibly break compatibility with some tooling

Reply:
>modifying the ABI/interface and possibly break compatibility with some tooling

Is that something we consider breaking? We've changed event name arguments in the past.

Reply:
All of the comparisons against `block.number` is replaced with the clock, so no problem with the naming, it doesn't change the interface.